### PR TITLE
Patch KB456535

### LINKED ISSUE #2: Unable to launch demo application after patching RyTek libraries in Windows 10 update KB4565351
dapplicationcrash

**Description:** Attempting to launch the test application directly from RyTek AFP program after patching the system.
**Windows Version:** Windows 10 Pro (64 bit)
**Latest Update:** KB4565351

### PR DESCRIPTION
**Pull request by RyanWalpole**
Implemented new libraries, scripts and updated documentation for the patch for the Windows 10 update KB4565351.

See issue #2

### CODE REVIEW

**Comment by RWELabs** [comment]
This pull request has been submitted for testing on Windows 10 update KB4565351 in sandbox environment.

File: README.md [hunk 1]
```diff
@@ -7,6 +7,9 @@
 
  ### Important!
  The RyTek3 platform was discontinued in 2017 with the last security patches being released by Ryan Walpole Enterprises in 2018. Software that uses the RyTek3 framework may be susceptible to malicious content and may not be secure to use. Please keep this in mind while using this framework patch.
+ 
+ #### Latest Windows Update
+ If you're using the latest security patch by Microsoft on your Windows 10 device, there is a chance that this has further broken the framework. Please be aware that if you're running the absolute latest in updates, you may have a different experience with the Application Framework Patch. For example, Windows 10 patch update KB4565351 was released on August 11 and broke several libraries. This may happen or not happen with any number of updates made by Microsoft. We do not guarantee that we will continue patching each security update if they are frequent.
```
Comment (RWELabs) [inline README.md]: Possible line breaks to make more sense.

**Review by RWELabs — COMMENTED** [review]
Pull request looks like it is completely compatible, however testing of the application on affected versions needs to be undertaken before a merge can be permitted.

**Comment by RyanWalpole** [comment]
Readme has been simplified.

**Review by RWELabs — APPROVED** [review]
Much better worded, makes a lot more sense.

**Comment by RWELabs** [comment]
Has been tested on target PC configuration. Can merge.

**Pull request merged**